.github/workflows/test-and-publish.yaml

```diff
@@ -0,0 +1,60 @@
+name: Publish to PyPI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.event.pull_request.merged == true
```

Comment:
What does it do?

Reply:
only runs if it the result of a push from a merged pull request, but it is unnecessary as our other branch protection rules guards from this anyway. Ill remove 👍🏻

Reply:
this will now run on any push to master, directly and through PR. Since we can control whether or not we want pushing to master to be possible with branch protection rules that might be a better place to handle this 👍🏻 So I deleted it 👍🏻